web/src/components/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from "react";
+
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Theme,
+  Button,
+} from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+
+interface Props {
+  showModal: boolean;
+  closeModal: () => void;
+  title: string;
+  onSubmit: () => void;
+}
+
+export const ConfirmationModal: FC<Props> = ({
```

Comment:
I'd stick to `ConfirmationDialog` name to keep consistency with used mui components